File: Selection-sort/cs/SelectionSort.cs
using System;
class BubbleSort
{
    static void selectionSort(int[] arr, int n)
    {
        for(int i = 0; i<n - 1;i++){
            int min = i;
            for(int j = i + 1; j< n;j++){
                if (arr[j] < arr[min]) {
                    min = j;
                }
            }
            int temp = arr[min];
            arr[min] = arr[i];
            arr[i] = temp;
        }
    }
    static void Main(string[] args)
    {
        int[] arr = {64, 34, 25, 12, 22, 11, 90};
        Console.WriteLine("Unsorted array:");
        for (int i = 0; i <arr.Length; i++)
            Console.Write(arr[i] + " ");
        Console.WriteLine();
        selectionSort(arr, arr.Length);
        Console.WriteLine("Sorted array:");
        for (int i = 0; i <arr.Length; i++)
            Console.Write(arr[i] + " ");
        Console.WriteLine();
    }
}

File: Selection-sort/js/SelectionSort.js
function selectionSort(arr, n) {
    for(let i = 0; i<n - 1;i++){
        let min = i;
        for(let j = i + 1; j< n;j++){
            if (arr[j] < arr[min]) {
                min = j;
            }
        }
        let temp = arr[min];
        arr[min] = arr[i];
        arr[i] = temp;
    }
}

let arr = [64, 34, 25, 12, 22, 11, 90];
console.log("Unsorted array:");
for (let i = 0; i < arr.length; i++) {
    process.stdout.write(arr[i] + " ");
}
console.log();
selectionSort(arr, arr.length);
console.log("Sorted array:");
for (let i = 0; i < arr.length; i++) {
    process.stdout.write(arr[i] + " ");
}
console.log();